microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,185 @@
 ---
-title: Windowed vs. visual hosting of WebView2
-description: Deciding whether to have your app use windowed hosting vs. visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
+title: Windowed vs. Visual hosting of WebView2
+description: Deciding whether to have your app use Windowed, Window to Visual, or Visual hosting of the WebView2 control.  Hosting WebView2 in windowed or visual environments.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: webview
-ms.date: 10/24/2022
+ms.date: 09/23/2024
 ---
-# Windowed vs. visual hosting of WebView2
+# Windowed vs. Visual hosting of WebView2
 
-There are two options for hosting the Microsoft Edge WebView2 control in your app: windowed hosting and visual hosting.
+There are three options for hosting the Microsoft Edge WebView2 control in your app:
+* [Windowed hosting: For displaying content quickly and easily](#windowed-hosting-for-displaying-content-quickly-and-easily).
+* [Window to Visual hosting: For a similar experience as Windowed hosting, with added benefits and tradeoff](#window-to-visual-hosting-for-a-similar-experience-as-windowed-hosting-with-added-benefits-and-tradeoff).
+* [Visual hosting: For more granular control over layout](#visual-hosting-for-more-granular-control-over-layout).
 
-If you use windowed hosting, which is a good starting point for most apps, you don't need to read this article.  If you want to provide a more custom user experience (UX), and want to use visual hosting, read this article.
+You don't need to read this article if you use Windowed hosting in most scenarios.  Windowed hosting is a good starting point for most apps.  Read this article:
+* If you are using Windowed hosting in uncommon scenarios and are experiencing persistent issues with DPI and scaling.  In that case, consider Window to Visual hosting.
+* If you want to provide a more custom user experience (UX) and want to use Visual hosting.
+
+The different approaches for hosting the WebView2 control in your app are similar in functionality, but they suit different needs depending on the app requirements, as follows:
 
 | Approach | Description | Optimized for |
 |---|---|---|
 | Windowed hosting | The WebView2 control takes input from the operating system (OS).  The OS sends the input to the WebView2. | Displaying web content quickly and easily, without having to include features for inputs, outputs, and accessibility. |
+| Window to Visual hosting | A combination of Windowed and Visual hosting. Similar to Windowed hosting except that WebView2 content is output to a visual that is hosted in a window rather having content output to the window directly. | A developer experience nearly identical to Windowed hosting, but with improved DPI/scaling handling and the caveat that pen input and handwriting is unsupported. |
```

Comment:
re: pen:
* change to "Windows Shell Handwriting (pen input)"?
* probably pen input is generally supported, but the "Shell Handwriting" experience (internal: Abydos) is not.
* were there other pen input scenarios that might be unsupported?
* apply same wording at "Shell Handwriting" below.

Reply:
updated this to be Windows Shell Handwriting